File: server/model/Owner.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/sequelize');

const Owner = sequelize.define('Owner', {
    id: {
        type: DataTypes.BIGINT,
        primaryKey: true,
        autoIncrement: true
    },
    name: {
        type: DataTypes.STRING(50),
        allowNull: false
    },
    lastName: {
        type: DataTypes.STRING(100),
        allowNull: false
    },
    documentType: {
        type: DataTypes.STRING(20),
        allowNull: false
    },
    documentNumber: {
        type: DataTypes.STRING(50)
    },
    phone: {
        type: DataTypes.STRING(30),
        allowNull: false,
        unique: true
    },
    address: {
        type: DataTypes.STRING(200),
        allowNull: false
    },
    city: {
        type: DataTypes.STRING(50),
        allowNull: false
    },
    postalCode: {
        type: DataTypes.STRING(10),
        allowNull: false
    },
    notes: {
        type: DataTypes.TEXT,
        allowNull: true,
        defaultValue: null
    }
}, {
    timestamps: true,
    paranoid: true
}
);
module.exports = Owner;

File: backend/src/main/java/com/danaslo/backend/owner/Owner.java
package com.danaslo.backend.owner;

public class Owner {
    private Long id;
    private String name;
    private String lastName;
    private String documentType;
    private String documentNumber;
    private String phone;
    private String address;
    private String city;
    private String postalCode;
    private String notes;

    public Owner(){}

    public Owner(String name, String lastName, String documentType,
                 String documentNumber, String phone, String address,
                 String city, String postalCode, String notes) {
        this.name = name;
        this.lastName = lastName;
        this.documentType = documentType;
        this.documentNumber = documentNumber;
        this.phone = phone;
        this.address = address;
        this.city = city;
        this.postalCode = postalCode;
        this.notes = notes;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getDocumentType() {
        return documentType;
    }

    public void setDocumentType(String documentType) {
        this.documentType = documentType;
    }

    public String getDocumentNumber() {
        return documentNumber;
    }

    public void setDocumentNumber(String documentNumber) {
        this.documentNumber = documentNumber;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getPostalCode() {
        return postalCode;
    }

    public void setPostalCode(String postalCode) {
        this.postalCode = postalCode;
    }

    public String getNotes() {
        return notes;
    }

    public void setNotes(String notes) {
        this.notes = notes;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
}
